addons/slideshow/functions/fnc_mapImage.sqf

```diff
@@ -5,12 +5,12 @@
  * Needs to be run on all machines
  *
  * Arguments:
- * 0: Position <ARRAY>
- * 1: Scale (1.0 fits entire map in x-dimension) <SCALAR>
- * 2: Markers [_pos, _text, _icon, _color] <ARRAY>
- * 3: MapType (0: Normal, 1: Topographic, 2: Satelite) or any custom class (even mission config) <SCALAR><STRING>
- * 4: Code to run on init (passed [_map, _display, _displayID]) <CODE>
- * 5: Resolution <SCALAR>
+ * 0: Position <ARRAY> (default: center of map)
+ * 1: Scale (1.0 fits entire map in x-dimension) <SCALAR> (default: 1.25)
+ * 2: Markers (each _x = [_pos, _text, _icon, _color]) <ARRAY> (default: [])
```

Comment:
Split this array out into multiple lines.

Reply:
requires https://github.com/acemod/ACE3/pull/9303 😄